File: Php/poo/figure/library/circle.php
<?php

        class circle extends figureGeometric{
        
            private float $radius;
            private float $calculate;

        public function __construct(float $radius){
            parent::__construct("Circle");
            $this->radius = $radius;
        }

        public function getRadius(): float {
            return $this->radius;
        }
    
        public function calculateArea(): float {
            $this->calculate = Pi() * pow($this->radius,2);
            return $this->calculate;
        }
    }

File: Java/poo/figure/library/circle.java
package library;

public class circle extends figureGeometric {
    private double radius;

    public circle(double radius){
        super("Circulo");
        this.radius = radius;
    }

    public double getRadius() {
        return this.radius;
    }
    @Override
    public double calculateArea(){
        return Math.PI * radius * radius;
    }
}
